modules/client/test/integration/tokenVoting-client/methods.test.ts

```diff
@@ -1442,6 +1447,54 @@ describe("Token Voting Client", () => {
         const token = await client.methods.getToken(pluginAddress);
         expect(token).toBe(null);
       });
+
+      it("Should check if a ERC20 is compatible with governance and throw", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new TokenVotingClient(ctx);
+        const erc20Token = await deployErc20();
+        expect(() =>
+          client.methods.isTokenGovernanceCompatible(
+            erc20Token.address,
+          )
+        ).rejects.toThrow(ERC165NotSupportedError)
```

Comment:
An ERC20 token will be wrapped, even if it doesn't support ERC-165.
The function should not throw an error in this case but say "needsWrapping".